Modules/Bibliographic/PRIVACY.md

```diff
@@ -0,0 +1,28 @@
+# Bibliograpghic Privacy
+Disclaimer: This documentation does not warrant completeness or correctness. Please report any missing or wrong information using the [ILIAS issue tracker](https://mantis.ilias.de) or contribute a fix via [Pull Request](docs/development/contributing.md#pull-request-to-the-repositories).
+
+## General Information
+- The Bibliographic module imports its data from a bibliography file. This data may or may not be personal in nature.
+- The following sections regarding the storage, presentation, deletion and export of personal information are written with the assumption that personal data has been imported. Otherwise there would be no handling of privacy-relevant data to describe at all.
+
+## Data being stored
+- Potentially personal data from the imported bibliography file is stored as "value".
```

Comment:
please delete. 
The Object service stores the account which created the object as it's owner and creation and update timestamps for the object.
[AccessControl](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/Services/AccessControl/PRIVACY.md)
[Info Screen Service](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/Services/InfoScreen/PRIVACY.md)